src/components/views/donate/DonateModal.tsx

```diff
@@ -174,12 +173,10 @@ const DonateModal: FC<IDonateModalProps> = props => {
 									delayedCloseModal(firstHash, secondHash);
 								return;
 							}
-							console.log('SecondDonation Success', secondHash);
 							setSecondTxStatus(EToastType.Success);
 							isSaved && delayedCloseModal(firstHash, secondHash);
 						})
 						.catch(({ txHash: secondHash }) => {
-							console.log('SecondDonation Error', secondHash);
```

Comment:
@divine-comedian We still have some bugs on donation page, I suggest we keep logs related to DonateModal component